doc/samples/readme-multi-transcode_linux.md

```diff
@@ -102,6 +102,7 @@ ParFile is extension of what can be achieved by setting pipeline in the command
 |  -q <quality\>| Quality parameter for JPEG encoder; in range [1,100], 100 is the best quality|
 |-l <numSlices\>|  Number of slices for encoder; default value 0|
 |-mss <maxSliceSize\>|Maximum slice size in bytes. Supported only with -hw and h264 codec. This option is not compatible with -l option.|
+|-BitrateLimit:<on,off>| Turn this flag ON to set bitrate limitations imposed by the SDK encoder: if target bitrate is lower than suggested by the SDK encoder, it will be changed to suggested. The default value is OFF, i.e. bitrate is not limited.|
```

Comment:
Modify bitrate to be in the range imposed by the SDK encoder. Setting this flag off may lead to violation of HRD conformance. The default value is OFF, i.e. bitrate is not limited.|

Reply:
changed